test/client_certs/generate.sh

```diff
@@ -10,7 +10,7 @@ cd "$(dirname "$0")"
 if [[ ! -f consul-agent-ca-key.pem ]] || [[ ! -f consul-agent-ca.pem ]]; then
     echo "Regenerating CA..."
     rm -f consul-agent-ca-key.pem consul-agent-ca.pem
-    consul tls ca create
+    consul tls ca create -days 36500
```

Comment:
The consul CLI updates are the important bits, because they always defaulted to 1 year. All of the openssl commands had this set to something like 10 years.